### PR TITLE
Feature/nex 2059/svgz support

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/NEX-2059

works together with https://github.com/oat-sa/extension-tao-itemqti/pull/1557

### CODE REVIEW

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
Tested on my local machine

File: models/classes/websource/BaseWebsource.php [hunk 1]
```diff
@@ -145,8 +145,9 @@ public function getMimetype($filePath)
                     }
                     break;
                 case 'svg':
+                case 'svgz':
                     //when there are more than one image in svg file - finfo recognizes it as `image/svg`, while it should be `image/svg+xml` or at least `text/plain` for a previous hack to work
-                    if ($mimeType === 'text/plain' || $mimeType === 'image/svg') {
+                    if ($mimeType === 'text/plain' || $mimeType === 'image/svg' || $mimeType === 'application/x-gzip') {
```
Comment (gabrielfs7) [inline models/classes/websource/BaseWebsource.php:150]: I think it reached a point we should have a constant called `ALLOWED_SVGZ_MIMETYPES` and just perform an `in_array` on this. I.e:

```suggestion
                    if (in_array($mimeType, self::ALLOWED_SVGZ_MIMETYPES)) {
```

So if we extend the list, the constant is updated. Not required, but an idea.

**Review by gabrielfs7 — APPROVED** [review]
Code LGTM, just an improvement idea

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: models/classes/websource/BaseWebsource.php (hunk 1 above)
Comment (mike-ionut-mihai-sandu-tao) [inline models/classes/websource/BaseWebsource.php:150]: let's create a ticket for unit testing this please

**Review by siwane — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Pull request merged**